src/super_gradients/common/registry/registry.py

```diff
@@ -13,6 +13,8 @@
 from super_gradients.training.models.segmentation_models.unet.unet_encoder import BACKBONE_STAGES
 from super_gradients.training.models.segmentation_models.unet.unet_decoder import UP_FUSE_BLOCKS
 from super_gradients.training.datasets.all_target_generators import ALL_TARGET_GENERATORS
+from super_gradients.training.datasets.all_collate_functions import ALL_COLLATE_FUNCTIONS
+from super_gradients.training.datasets.samplers.all_samplers import SAMPLERS
```

Comment:
please add the native torch samplers.
look for all classes that inherit from `Sampler` in `torch.utils.data.sampler`

Reply:
added in commit "updated_ fixes"